frontend/src/Pages/components/AdminCategories/AddCategory.tsx

```diff
@@ -0,0 +1,65 @@
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router-dom";
+
+const axios = require("axios").default;
+
+const AddCategory = () => {
+  const history = useHistory();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit = async (data: { title: string; description: string }) => {
+    await axios.post("http://127.0.0.1:8000/api/quizzes", data);
```

Comment:
For this  anyting na request related its cleaner if i separatete nimo sa API js files nimo instead para compile tanan nimo api calls ngadto. Also for documentation na pd of all your available api function. other developers make also know what api calls are already implemented.

 So, put this in your API js file
then import that api and call the method like i taught in previous pr.

```
saveSomething(data).then(res => {
   /** Handle Success */
}.catch(err => {
  console.error(err);
  /** Handle Fail */
});